test/push-shared-code-with-vars/in/my-branch/transformation/keboola.python-transformation-v2/transformation-with-shared-code/blocks/001-block-1/001-shared-code-used/code.py

```diff
@@ -1 +1 @@
-{{123456}}
+# {{:codes/code-with-variables}}
```

Comment:
Ukazem to najprv na testoch:
- namiesto ide sa v transformation code blocks uklada relativna cesta ku shared code config row, tj. cesta je v `code.py`.
- cesta ku shared code config, je v `conifg.json` (to uz bolo):
https://github.com/keboola/keboola-as-code/blob/f98a1d60cc34295dfb3bd3db9792f8c020be95f1/test/push-shared-code-with-vars/in/my-branch/transformation/keboola.python-transformation-v2/transformation-with-shared-code/config.json#L3
- cela cesta je potom `{branch}/_shared/keboola.python-transformation-v2/codes/code-with-variables`